prose.md

```diff
@@ -288,18 +289,20 @@ which is a big improvement over the current system.
 > timing 2021-06-09: ~ 1 minute
 > timing 2021-06-10: ~ 1 minute
 
-We have just barely finished with the alpha testing and the next phase for us is
-to address all the questions and concerns that were brought up (for example: how
+We have just finished with this first testing phase and the next step for us is
+to address the questions and concerns that were brought up (for example: how
 do I use this without clobbering the current R installation on my system?),
-clean up documentation, and build it for the beta release, where we will try it
-on a few live lessons to make sure this works for the community. It's a slow
+improve the documentation, and get ready for the beta release, where we will try it
+on a few live lessons to identify painpoints for the community. It is a slow
 process, but this way we can avoid major unforseen issues (minor issues are a
 given), bring users in on the ground floor, get valuable feedback, and
 strengthen trust with our community.
 
+> if you want to go fast, go alone; if you want to go far, go together
```

Comment:
I was low-key looking to avoid this quote partially because it's cheesy, but also because there is an aspect of this that's only partially true of this project. Yes, we are gathering community feedback for it, but the vast majority of the project was written by myself.